src/apps/testapps/testH3ToLocalIj.c

```diff
@@ -66,22 +66,22 @@ void h3ToLocalIj_coordinates_assertions(H3Index h3) {
 
     CoordIJ ij;
     t_assert(H3_EXPORT(experimentalH3ToLocalIj)(h3, h3, &ij) == 0,
-             "failed to get ij");
+             "get ij for origin");
     CoordIJK ijk;
     ijToIjk(&ij, &ijk);
     if (r == 0) {
-        t_assert(_ijkMatches(&ijk, &UNIT_VECS[0]) == 1, "not at 0,0,0 (res 0)");
+        t_assert(_ijkMatches(&ijk, &UNIT_VECS[0]) == 1, "res 0 cell at 0,0,0");
     } else if (r == 1) {
         t_assert(_ijkMatches(&ijk, &UNIT_VECS[H3_GET_INDEX_DIGIT(h3, 1)]) == 1,
-                 "not at expected coordinates (res 1)");
+                 "res 1 cell at expected coordinates");
     } else if (r == 2) {
         CoordIJK expected = UNIT_VECS[H3_GET_INDEX_DIGIT(h3, 1)];
         _downAp7r(&expected);
         _neighbor(&expected, H3_GET_INDEX_DIGIT(h3, 2));
         t_assert(_ijkMatches(&ijk, &expected) == 1,
-                 "not at expected coordinates (res 2)");
+                 "res 2 cell at expected coordinates");
     } else {
-        t_assert(0, "wrong resolution");
+        t_assert(0, "resolution supported by test function (coordinates)");
```

Comment:
Nit: This (essentially `assert.fail`) is the one place where negative assertion messages actually make more sense IMO.